### DISCUSSION TITLE
Allow the use of more placeholders in Task Signup

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#312] [Ideas]
I tried to use #_PHONE and #_COUNTRY in a template for Task Signup but they just displayed the code as shown in the screenshot.

Nowadays, we use phone messengers to communicate so we need the volunteers' phone numbers. Country is also necessary as without it, we can't add the volunteers to the messengers.

Also, #_GDPR is not allowed. Shouldn't this be necessary when we are getting personal info from the volunteers?

![Screenshot 2023-10-07 171655](https://github.com/liedekef/events-made-easy/assets/101639308/9aeb50e8-501f-4481-99e6-5054d529064b)

**Comment by liedekef** [comment]
PHONE is weird, that should just work (and it works fine here) ==> what version are you on?
Next to that: the next version will allow more fields for a volunteer (all person-related fields in fact, and #_GDPR too)

**Reply by refugeindhamma** [reply]
Version 2.4.9

Thanks for the update for next version.

**Reply by liedekef** [reply]
But still: #REQ_PHONE should work fine (I tried that here). Is it possible there was a character behind it? The regex had an error there ...

**Reply by refugeindhamma** [reply]
This is the code in the template:
```
<tr>
<td style="padding:10px"><strong>Email<span style="color:red">*</span></strong><br />#REQ_EMAIL{Eg: johndoh@email.com}</td>
<td style="padding:10px"><strong>Handphone<span style="color:red">*</span></strong><br />#REQ_PHONE{+100 12345678}</td>
</tr>
```

**Reply by liedekef** [reply]
Yeah, due to #REQ_PHONE{+100 12345678} it won't work, just #REQ_PHONE will work ok (without the extra phonenumber info). The dev-version fixed that already.
